recipes/gpm-imerg/recipe.py

```diff
@@ -0,0 +1,32 @@
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern
+import pandas as pd
+
+#TODO: replace with ENV vars
+username = password = 'pangeo@developmentseed.org'
+
+def make_filename(time):
+    input_url_pattern = (
+        "https://arthurhouhttps.pps.eosdis.nasa.gov/gpmdata/{yyyy}/{mm}/{dd}/imerg/3B-HHR.MS.MRG.3IMERG.{yyyymmdd}-S{sh}{sm}00-E{eh}{em}59.{MMMM}.V06B.HDF5"
+    ).format(
+        yyyy=time.strftime("%Y"),
+        mm = time.strftime("%m"),
+        dd = time.strftime("%d"),
+        yyyymmdd=time.strftime("%Y%m%d"),
+        sh = time.strftime("%H"),
+        sm = time.strftime("%M"),
+        eh = time.strftime("%H"),
+        em = (time+pd.Timedelta("29 min")).strftime("%M"),
+        MMMM = f'{(time.hour*60 + time.minute):04}'
+    )
+    return input_url_pattern
+
+time_concat_dim = ConcatDim("time", dates, nitems_per_file=1)
```

Comment:
The recipe test caught an error here: https://github.com/pangeo-forge/staged-recipes/runs/3558398362?check_suite_focus=true#step:7:43

You need to define `dates` as a sequence, e.g. via `pd.date_range`.

Reply:
Apologies, missed that in copy/pasting from my test notebook. Should be fixed now.

Reply:
@wildintellect It appears you are also missing the define for `aiohttp` https://github.com/pangeo-forge/staged-recipes/runs/3559506426?check_suite_focus=true#step:7:41